autorag/data/qacreation/llama_index.py

```diff
@@ -8,6 +9,9 @@
 
 from autorag.utils.util import process_batch
 
+
+nest_asyncio.apply()
```

Comment:
Recommend to delete this if you add this for jupyter notebook.
Unwanted nest asyncio at non-jupyter environment.